### PR TITLE
Add 1.11.0

### PR DESCRIPTION
**Pull request by hexonaut**

### CODE REVIEW

File: releases/mainnet/1.11.0/contracts.json [hunk 1]
```diff
@@ -0,0 +1,306 @@
+{
+    "CHANGELOG": "0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F",
```
Comment (naszam) [inline releases/mainnet/1.11.0/contracts.json:2]: extra spacing, line and missing flash killer ([diffcheck](https://www.diffchecker.com/rMHNsIH0))

File: releases/mainnet/1.11.0/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+			<ul>
+				<li><a href="/releases/mainnet/1.11.0/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmVvhd31mz3L92gA1R2qR4SQFLJUyntXPbAAXdedoKn7Jf">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.11.0/abi/index.html">ABIs</a></li>
+
+				<li>Links to GitHub repositories
+					<ul>
+						<li>DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.11.0" target="_blank">changelog</a>)</li>
+					</ul>
+				</li>
```
Comment (naszam) [inline releases/mainnet/1.11.0/index.html:20]: I think this block should be commented for now as `dss-deploy-scripts` `1.11.0` have not been released yet

File: releases/goerli/1.11.0/index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<html>
+	<head>
+		<title>Goerli Release 1.11.0.</title>
```
Comment (naszam) [inline releases/goerli/1.11.0/index.html:3]: extra dot (same for L8, L12, L14)
```suggestion
		<title>Goerli Release 1.11.0</title>
```

**Comment by naszam** [comment]
I think we might want to consider replacing `Flapper.abi` in `dss-deploy` (or different folder eventually) in `(mainnet/goerli)_abi_1.11.0.zip` with the updated abi, extracted from ([flap-rate-limit](https://github.com/makerdao/dss/pull/244)).
cc: @brianmcmichael 
Also, noticed a `__MACOSX` artefact unzipping `goerli_abi_1.11.0.zip` that I think was introduced in `1.10.1` and not present in `1.10.0`
cc: @gbalabasquer

**Review by naszam — COMMENTED** [review]
Checked:

- Goerli `1.11.0`
    - root `index.html` Looks Good
    - `1.11.0` `index.html` (extra dots)
    - `contracts.json` Looks Good (diffed against `1.10.1`)
    - IPFS Hash Match `contracts.json`
    - Changelog Change Looks Good (New Flap, No `CRVV1ETHSTETH` deployments on Goerli)
    - ABIs (macos artifacts, old Flap abi, diffed against `1.10.1` (matches), diffed `index.html`)
    - Active Links Updated (`1.11.0`)
- Mainnet `1.11.0`
    - root `index.html` Looks Good
    - `1.11.0` `index.html` Looks Good
    - `contracts.json` Looks Good (diffed against `1.10.1`)
    - IPFS Hash Match `contract.json`
    - Changelog Changes Look Good (New Flap, `CRVV1ETHSTETH` Collateral Onboarding Deployments)
    - ABIs (no artifacts, old Flap abi, diffed against `1.10.1` (`dss-crop-join`), diffed `index.html`)
    - Active Links Update (`1.11.0`, pending merging master)

File: releases/goerli/1.11.0/index.html (hunk 3 above)
Comment (hexonaut) [reply releases/goerli/1.11.0/index.html:3]: oops

**Review by naszam — APPROVED** [review]
Checked:

- `1.11.0` goerli `index.html` Looks Good 
- macos artifacts removed in `1.10.1` and goerli `1.11.0`
- `Flapper.abi` replaced in `dss-deploy` in ABIs on mainnet and goerli `1.11.0` (`flap-rate-limit`)

LGTM

**Review by zdumitru — APPROVED** [review]

**Pull request merged**